XRTK.Oculus/Packages/com.xrtk.oculus/Editor/AndroidManifest.OVRSubmission.xml

```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
Why has this file been removed?  It is critical to the Oculus platform to apply a standard Android manifest for operation on the quest.

All these files that have been deleted are critically required

Reply:
It wasn't removed. it was moved into a different folder

Reply:
Ah figured it out. The new folder name was one of the ignored types. Renamed and re added these files correctly now.